Projects/TerminateContracts/Sources/Views/TerminationSummaryScreen.swift

```diff
@@ -86,60 +102,38 @@ struct TerminationSummaryScreen: View {
     private var addonContent: (some View)? {
```

Comment:
No need for `(some View)?` here - simply `some View` would do.
Also
`    @ViewBuilder`
is unnecessary since we are returning VStack, not something that can vary